src/app/GitCommands/Remotes/ConfigFileRemoteSettingsManager.cs

```diff
@@ -147,7 +147,25 @@ bool IsSettingForBranch(string setting, string branchName)
                                    .Select(t => new GitRef(module, null, t[1]))
                                    .FirstOrDefault(h => h.IsHead);
 
-            return remoteHead?.Name;
+            if (remoteHead is not null)
+            {
+                return remoteHead.Name;
+            }
+
+            bool IsSettingForAnyBranch(string setting)
```

Comment:
Move inline functions behind the `return` statement, please.

Reply:
➡️ Done